etc/skydive.yml.default

```diff
@@ -104,6 +104,8 @@ agent:
     # stats_update: 1
   metadata:
     info: This is compute node
+  tcp_len_by_seq: true
```

Comment:
My patch doesn't deal so much with retransmissions. It refers more to the case when for example we capture only partial traffic (such as SYN/FIN) and then using sequence numbers we calculate traffic length. 
Thing that you propose looks like good thing to have, but I see it as rather big and independent feature. So, I propose opening it as a new open issue and starting discussing it via our meetings.